index.bs

```diff
@@ -4557,6 +4557,12 @@ macros:
 The number of channels of the output always equals the number of
 channels of the AudioBuffer assigned to the {{AudioBufferSourceNode/buffer}} attribute, or is one
 channel of silence if {{AudioBufferSourceNode/buffer}} is <code>null</code>.
+In addition, the {{AudioBufferSourceNode}} must output a single
+channel of silence when the end of the
```

Comment:
Perhaps s/when/after/ to add a hint that this is not in the middle of a rendering quantum.